api/identityprovider_suite_test.go

```diff
@@ -40,6 +40,8 @@ var _ = Describe("Process Strategy: ", func() {
 	It("create a Strategy CR", func() {
 		cr := sampleStrategy.DeepCopy()
 		createdCR, err := clientSet.IdentitatemV1alpha1().Strategies(cr.Namespace).Create(context.TODO(), cr, metav1.CreateOptions{})
+		cr.Spec.RemediationType = identitatemv1alpha1.InformRemediationActionType
```

Comment:
This has no effect as they are set after the creation, maybe you wanted to check if I change it after I want to make sure nothing changed on the saved CR.

Reply:
Right.  Was part of debugging to fix the issue.  I added this code before I realized the CR was being read from a yaml file.  Removed lines.